source/includes/_fiat_to_crypto_flow.md

```diff
@@ -0,0 +1,236 @@
+# Fiat to Crypto flow
+
+This section describes the endpoints used in the fiat to crypto flow. 
+
+## Create Fiat to Crypto Invoice
+
+> Example Call
+
+```shell
+curl -X POST 'https://api.onramp.ltd/rpc/create_fiat_to_crypto_invoice' \
+-H 'Content-Type: application/json' \
+-H 'x-xco-authorization: Bearer 00000000-0000-0000-0000-000000000000' \
+-d '{
+    "fiat_amount": 3000,
+    "fiat_currency": "EUR",
+    "callback_url": "https://www.example.com",
+    "timeout_in_sec": 3600,
+    "offer_skin": {
+        "title": "The Nice Merchant",
+        "image": "https://static.example.com/merchant-logo",
+        "description": "Awesome green Pluff with handler"
+    },
+    "billing_details": {
+        "payer_email": "example@example.com",
+        "payer_first_name": "John",
+        "payer_last_name": "Doe",
+        "payer_phone": "0034666444446",
+        "birth_date": "1980-11-24 00:00:00.000000+00",
+        "street": "Main Street",
+        "unit": "Floor 7 12B",
+        "postal_code": "0000001",
+        "city": "Big Town",
+        "county": "Your County",
+        "state": "CA",
+        "prefecture": "Prefecture-ku",
+        "country": "GBR",
+        "kyc_verified": "2020-01-01 12:45:00.000000+00",
+        "kyc_document": "Passport",
+        "kyc_reference": "25177aa2-d848-846d-226c-97ec3096f5fe",
+        "address_verified": "2020-01-01 12:55:00.000000+00",
+        "address_doc_reference": "3sdffdss-4mgo-44gfd-34dx-r34rfdfs4gf",
+        "player_level": "New",
+        "member_since": "2020-01-01 12:35:00.000000+00",
+        "merchant_customer_id": "u2340112"
+    },
+    "card_number": "4929421234600821",
+    "card_expiry_date": "2030-12-01",
+    "card_cvv": "356",
+    "card_holder_first_name": "John",
+    "card_holder_last_name": "Doe",
+    "card_type": "VISA",
+    "order_id": "order-1234",
+    "first_time": true,
+    "is_banking": false,
+    "crypto_data": {
+        "currency": "BTC",
+        "address": "1Lbcfr7sAHTD9CgdQo3HTMTkV8LK4ZnX71"
```

Comment:
@badborg @mxsur please confirm that this info is enough to do a trade

Reply:
LGTM